src/index.html

```diff
@@ -1,14 +1,63 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8"> </meta>
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    ></meta>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge"></meta>
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.css"> </link>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <a href="index.html">
+      <img class="logo" src="/images/logo.png" 
+          alt="mojo-logo">
+    </a>   
+    <nav class="nav">
+      <ul>
+        <li class="nav__list">
+          <a href="#" class="is-active">apple</a>
+        </li>
```

Comment:
Generally use `class="nav__list"` for `ul` tag
use `class="nav__item` for `li` tag

Reply:
understood